start

```diff
@@ -361,6 +362,12 @@ function init_stellar_core() {
   run_silent "finalize-core-config-run-local" perl -pi -e "s/__RUN_STANDALONE__/$RUN_STANDALONE/g" etc/stellar-core.cfg
   run_silent "finalize-core-config-manual-close" perl -pi -e "s/__MANUAL_CLOSE__/$ENABLE_CORE_MANUAL_CLOSE/g" etc/stellar-core.cfg
 
+  if [ "$NETWORK" = "standalone" ] && [ "${CORE_SUPPORTS_TESTING_SOROBAN_HIGH_LIMIT_OVERRIDE}" = "true" ]; then
```

Comment:
should this be checking for `NETWORK=local` now?

Reply:
Yes!

Reply:
Opened fix in https://github.com/stellar/quickstart/pull/493